stories/assets/js/BioCard_custom.js

```diff
@@ -0,0 +1,24 @@
+/* Bio JS start custom */
+export function BioCardFun() {
+var $modalOverlay = $('#myModal');
+var $modal        = $('.modal-content');
+var $modalClose   = $('.modal-close-button');
+var $modalOpen    = $('#bio-card');
+
+$modalOpen.click(function(){
+  $('#myModal').addClass('open');
+});
+
+$modalOverlay.click(function(){
+  $('#myModal').removeClass('open');
+});
+
+$modal.click(function(e) {
+   e.stopPropagation(); 
+});
+
+$modalClose.click(function(){
+  $modalOverlay.click();
+});
+}
+/* Bio JS end custom */
```

Comment:
@harishkumar-srijan please make this function generic that is modal and not biocardfunc and please pass the selectors as the parameters in functions